app/javascript/shared/components/FluentIcon/DashboardIcon.vue

```diff
@@ -33,6 +34,10 @@ export default {
       type: String,
       default: '0 0 24 24',
     },
+    isFluentIcon: {
```

Comment:
I would accept a `type` props defaults to `fluent`, can be anything else. It would give us the option to add more libraries to the same icon set.